### PR TITLE
initial docker image

### PR DESCRIPTION
**Pull request by Cobra1978**
Following this checklist to help us incorporate your 
contribution quickly and easily:

Initial Docker Image 

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [ X] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

This is the initial release.

Simply clone the repository, go in the directory herddb-docker and do a docker build to obtain the image.

### CODE REVIEW

File: herddb-docker/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# How to use this image
+
+## Running single instance HerdDB
```
Comment (eolivelli) [inline herddb-docker/README.md]: can you please add a paragraph about how to build the image ?

Comment (Cobra1978) [reply herddb-docker/README.md]: i'm working on the documentation, and will put those instruction on the next pull request.

**Comment by eolivelli** [comment]
Hello @Cobra1978
What is the status of this patch?

It would be so great to have a recipe to start HerdDB on Kubernetes pods.

**Comment by eolivelli** [comment]
@Cobra1978 what is the status of this patch ?
I am going to pick this up if you do not have cycles.

We should release it for 0.16

**Comment by Cobra1978** [comment]
Sorry but by now i don't have enough time to go on witth this issue, feel free to start from the work i'v made.

**Pull request closed without merging**